### PR TITLE
Adding code to handle callback

### PR DESCRIPTION
**Pull request by ankita10r**

### CODE REVIEW

**Review by avikganguly01 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -94,3 +94,15 @@ This API will return Bridge Identifier by which you can use while sending SMS
 		}]
 
 Note: While sending SMS, don't include country code as part of each message. This country code will be prefixed by taking value from provider details. You need configure host-address: [your public ip on which message-gateway is running] in application.yml
+
+#### To add Telerivet Java Client : Run the following commands 
+
+    git clone https://github.com/Telerivet/telerivet-java-client
+    mvn install
+
```
Comment (avikganguly01) [inline README.md:102]: @ankita10r : See JFrog URL posted in Group.

File: README.md [hunk 2]
```diff
@@ -94,3 +94,15 @@ This API will return Bridge Identifier by which you can use while sending SMS
...
+#### To add PHEE Connector Channel : Run the following commands
+
+    git clone https://github.com/openmf/ph-ee-connector-common
+    mvn instal
```
Comment (avikganguly01) [inline README.md:106]: @ankita10r : This dependency should not be part of message gateway. Please remove connector common dependency from message gateway.

Comment (ankita10r) [reply README.md:106]: ok

File: README.md (hunk 1 above)
Comment (ankita10r) [reply README.md:102]: ok

Comment (ankita10r) [reply README.md:102]: Resolved with PR https://github.com/openMF/message-gateway/pull/28

File: README.md (hunk 2 above)
Comment (ankita10r) [reply README.md:106]: Resolved with PR https://github.com/openMF/message-gateway/pull/28

**Pull request merged**